airbyte-integrations/bases/debezium-v1-9-2/src/main/java/io/airbyte/integrations/debezium/internals/FilteredFileDatabaseHistory.java

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.integrations.debezium.internals;
-
-import io.debezium.config.Configuration;
-import io.debezium.relational.history.AbstractDatabaseHistory;
-import io.debezium.relational.history.DatabaseHistoryException;
-import io.debezium.relational.history.DatabaseHistoryListener;
-import io.debezium.relational.history.FileDatabaseHistory;
-import io.debezium.relational.history.HistoryRecord;
-import io.debezium.relational.history.HistoryRecord.Fields;
-import io.debezium.relational.history.HistoryRecordComparator;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.util.function.Consumer;
-
-/**
- * MySQL Debezium connector monitors the database schema evolution over the time and stores the data
- * in a database history file. Without this file we can't fetch the records from binlog. We need to
- * save the contents of the file. Debezium by default uses
- * {@link io.debezium.relational.history.FileDatabaseHistory} class to write the schema information
- * in the file. The problem is that the Debezium tracks the schema evolution of all the tables in
- * all the databases, because of that the file content can grow. In order to make sure that debezium
- * tracks only the schema of the tables that are present in the database that Airbyte is syncing, we
- * created this class. In the method {@link #storeRecord(HistoryRecord)}, we introduced a check to
- * make sure only those records are being saved whose database name matches the database Airbyte is
- * syncing. We tell debezium to use this class by passing it as property in debezium engine. Look
- * for "database.history" property in {@link DebeziumRecordPublisher#getDebeziumProperties()}
- * Ideally {@link FilteredFileDatabaseHistory} should have extended
- * {@link io.debezium.relational.history.FileDatabaseHistory} and overridden the
- * {@link #storeRecord(HistoryRecord)} method but {@link FilteredFileDatabaseHistory} is a final
- * class and can not be inherited
- */
-public class FilteredFileDatabaseHistory extends AbstractDatabaseHistory {
```

Comment:
The new debezium version filters out the schema history records only fir the database that its syncing so our custom logic is not required anymore